user/pages/04.Reference/05.object-storage/docs.en.md

```diff
@@ -49,6 +49,9 @@ Every bucket name can only be used once per SysEleven Stack region among all cus
 Basically, SysEleven Stack Object Storage is a big key/value store.
 A file or file object can be assigned a file name like key, and made available under this key.
 
+!! **Be aware**
+!! We recommend to avoid using a slash (`/`) as bucket or object name ending. The object storage derrives and emulates a folder structure from the object names and in consequence may interpret objects as folders which causes unwanted side effects such as objects/buckets being undeleteable or unaccessable. In the case you are stuck with such a phenomenon, please contact our [Cloud-Support (cloudsupport@syseleven.de)](../../06.Support/default.en.md).
```

Comment:
derrives?

Reply:
And actually, we disencourage the use of slashes in bucket names entirely.

Reply:
Updated that part again =)

Reply:
Using your suggestion, thanks for taking some time again to rephrase this whole part! <3